api/v1alpha6/openstackmachinetemplate_webhook_test.go

```diff
@@ -150,6 +150,7 @@ func TestOpenStackMachineTemplate_ValidateUpdate(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
This is fine, but backporting a PR which changes api/ is surprising.

It would be good to move most code out of api which isn't type definitions at some point.